src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -409,47 +396,17 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
 
-.v-icon {
+:deep(.v-icon) {
```

Comment:
Add comment indicating what you're doing (or why). This will help the next developer to look at this, and may help the upcoming upgrade from Vuetify 2 to 3.

Reply:
Well, yes, but I can see that from the code. The real question is what/why are you doing this? For example, "make icon taller so that it matches the text beside it" (or whatever).

Reply:
I removed my comment also removed my change of `:deep`, cause it doesn't need anymore after using slots for "help toggle" section. the `.v-icon` is not added by me, I think it's better to keep as original.

Reply:
ok

Reply:
PS If you do need to set this CSS, and it's in a child component, I suggest moving this CSS to the child component instead of using deep() here.

The idea here is "try to keep related things together."

Reply:
ok, thanks!